keyboards/rmi_kb/tkl_f13hs/info.json

```
@@ -0,0 +1,135 @@
+{
+    "keyboard_name": "TKL-F13HS",
+    "manufacturer": "RMI-KB",
+    "url": "",
+    "maintainer": "ramonimbao",
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "usb": {
+        "vid": "0xB16B",
+        "pid": "0x2125",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "rows": ["GP0",  "GP1",  "GP2",  "GP21", "GP3",  "GP4"],
+        "cols": ["GP29", "GP28", "GP27", "GP26", "GP25", "GP24", "GP23", "GP22", "GP19", "GP18", "GP17", "GP16", "GP15", "GP14", "GP13", "GP12", "GP6"]
+    },
+    "rgb_matrix": {
+        "driver": "WS2812",
+        "layout": [
+            {"flags": 8, "matrix": [0, 15], "x": 211, "y": 0},
+            {"flags": 8, "matrix": [0, 13], "x": 182, "y": 0},
+            {"flags": 8, "matrix": [0, 0], "x": 0, "y": 0},
+            {"flags": 8, "matrix": [3, 0], "x": 2, "y": 27}
+        ]
+    },
```

migrated from `rules.mk`
```suggestion
    },
    "ws2812": {
        "driver": "vendor"
    },
```